consensus/builddev.go

```diff
@@ -14,6 +14,7 @@ const (
 	TargetWindow          = BlockHeight(80)        // Number of blocks to use when calculating the target.
 	MedianTimestampWindow = 11                     // Number of blocks that get considered when determining if a timestamp is valid. Should be an odd number.
 	FutureThreshold       = Timestamp(3 * 60 * 60) // Seconds into the future block timestamps are valid.
+	SiafundCount          = Currency(10 * 1000)    // The total (static) number of siafunds.
```

Comment:
I tend to prefer "NumFoos` to `FooCount`. Maybe `TotalSiafunds` would be better here.